react/ModalTrigger.tsx

```diff
@@ -21,7 +21,9 @@ const ModalTrigger: React.FC<{}> = ({ children }) => {
   }
 
   return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
Whats this comment here?

Reply:
> Whats this comment here?

This comment is disabling the `click-events-have-key-events` rule from eslint, because doesn't make any sense to add a hotkey to a `ModalTrigger`